src/openrct2/peep/staff.c

```diff
@@ -1404,3 +1423,33 @@ bool staff_set_colour(uint8 staffType, colour_t value)
 	}
 	return true;
 }
+
+uint32 staff_get_available_entertainer_costumes()
+{
+	init_scenery();
+
+	uint32 entertainerCostumes = 0;
+	for (sint32 i = 0; i < 19; i++) {
```

Comment:
This stray 19 is to remind me that we should not be using `window_scenery_tab_entries` here and create a better scenery management service.